sql/src/test/java/io/crate/integrationtests/ArithmeticIntegrationTest.java

```diff
@@ -70,6 +72,14 @@ public void testSelectWhereArithmeticScalar() throws Exception {
         assertThat(response.rowCount(), is(1L));
         assertThat((Long) response.rows()[0][0], is(2L));
 
+        execute("select trunc(d) from t where trunc(d, 1) = 2.2");
+        assertThat(response.rowCount(), is(1L));
+        assertThat((Long) response.rows()[0][0], is(2L));
+
+        execute("select trunc(d) from t where trunc(d, 2) = 29.19");
```

Comment:
not sure if we really need this additional integration tests, I think the scalar unit tests are enough. Or do I miss some by unit tests not covered scenarios?

Reply:
I agree, the unit tests should cover all scenarios.